cmd/appliance/force_disable_controller.go

```diff
@@ -0,0 +1,361 @@
+package appliance
+
+import (
+	"bytes"
+	"context"
+	"errors"
+	"fmt"
+	"io"
+	"sort"
+	"strings"
+	"sync"
+	"text/template"
+
+	"github.com/AlecAivazis/survey/v2"
+	"github.com/appgate/sdp-api-client-go/api/v18/openapi"
+	appliancepkg "github.com/appgate/sdpctl/pkg/appliance"
+	"github.com/appgate/sdpctl/pkg/appliance/change"
+	"github.com/appgate/sdpctl/pkg/configuration"
+	"github.com/appgate/sdpctl/pkg/docs"
+	"github.com/appgate/sdpctl/pkg/factory"
+	"github.com/appgate/sdpctl/pkg/prompt"
+	"github.com/appgate/sdpctl/pkg/tui"
+	"github.com/appgate/sdpctl/pkg/util"
+	"github.com/hashicorp/go-multierror"
+	log "github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
+)
+
+type cmdOpts struct {
+	Config        *configuration.Config
+	Appliance     func(c *configuration.Config) (*appliancepkg.Appliance, error)
+	Out           io.Writer
+	SpinnerOut    func() io.Writer
+	NoInteractive bool
+	CiMode        bool
+}
+
+func NewForceDisableControllerCmd(f *factory.Factory) *cobra.Command {
+	opts := cmdOpts{
+		Appliance:  f.Appliance,
+		Config:     f.Config,
+		Out:        f.IOOutWriter,
+		SpinnerOut: f.GetSpinnerOutput(),
+	}
+	cmd := &cobra.Command{
+		Use:         "force-disable-controller [hostname...]",
```

Comment:
Yes, this can definitely be improved. The issue spec says that [hostname...] should be the argument, but the API needs the IDs, so the error message should show correct info. Also, we could accept ID:s as well.

Reply:
So this commit allows for both hostname and ID as arguments: 2359747b9177fec8deab4c94ca7bbf6e7bd40433

Reply:
the problem with a empty table and a false positive spinner still persist, for example:

```bash
sdpctl appliance force-disable-controller foobar

FORCE-DISABLE-CONTROLLER SUMMARY

This will force disable the selected controllers and announce it to the remaining controllers. The following Controllers are going to be disabled:

Name    Hostname    Status    Version
----    --------    ------    -------

? Do you want to continue? Yes
 ✓ disabling controller: done
2 errors occurred:
	* Validation failed.
	* applianceIds must not be empty
```

Reply:
fixed now as well